charts/elasticsearch/templates/master/es-master-statefulset.yaml

```diff
@@ -141,11 +141,13 @@ spec:
         - containerPort: {{ .Values.common.ports.transport }}
           name: transport
           protocol: TCP
-        # readinessProbe:
-        #   httpGet:
-        #     path: /_cluster/health?local=true
-        #     port: {{ .Values.common.ports.http }}
-        #   initialDelaySeconds: 5
+        readinessProbe:
+          httpGet:
+            # local: If true, the request retrieves information from the local node only.
+            # Defaults to false, which means information is retrieved from the master node.
+            path: /_cluster/health?local=true
+            port: {{ .Values.common.ports.http }}
+          initialDelaySeconds: 5
```

Comment:
The metrics exporter is dropping occasionally. In the logs, I have found it's getting 503 from the master check. I expect the master it's routed to is not ready, serving on the port. this will take the master out of the astronomer-elasticsearch service that metrics exporter queries when it's not serving HTTP. There were also timeout logs, however and this change does not address that aspect of the metrics exporter failing to get metrics.

Reply:
Previously, this was commented out. I suspect the initial reason for this might be not longer applicable with the addition of another change in this PR "publishNotReadyAddresses: true"